third_party/googletest/BUILD.gn

```diff
@@ -137,14 +152,44 @@ source_set("gtest") {
     deps += [ "//base" ]
   }
 
+  if (is_cobalt_hermetic_build) {
+    deps += [ "//starboard/common:common_headers_only" ]
+    public_deps = [ "//starboard:starboard_group" ]
+
+    # TODO: b/399507045 - Cobalt: Investigate removing this code after fixing build/runtime errors.
+    if (enable_cobalt_hermetic_hacks) {
+      sources -= [
+        "custom/gtest/internal/custom/chrome_custom_temp_dir.cc",
+        "custom/gtest/internal/custom/chrome_custom_temp_dir.h",
+        "custom/gtest/internal/custom/gtest.h",
+        "custom/gtest/internal/custom/gtest_port_wrapper.cc",
+        "custom/gtest/internal/custom/stack_trace_getter.cc",
+        "custom/gtest/internal/custom/stack_trace_getter.h",
+      ]
+      sources += [ "src/googletest/src/gtest-port.cc" ]
+      configs += [ "//build/config/compiler:chromium_code" ]
+
+      # Ideally we would want to remove
+      # //starboard/build/config:default_rtti from the list of configs too.
+      # Unfortunately GN doesn't provide us a way to remove things from
+      # dependents' configs list.
+      # It happens to be that all_dependent_configs get processed after default
+      # configs, so the -frtti flag comes after the -fno-rtti flag on the command
+      # line, so it happens to work in this case.
+      all_dependent_configs = [ "//build/config/compiler:rtti" ]
+    }
+  }
+
   if (is_fuchsia) {
     deps += [
       "//third_party/fuchsia-sdk/sdk/pkg/fdio",
       "//third_party/fuchsia-sdk/sdk/pkg/zx",
     ]
   }
 
-  if (gtest_enable_absl_printers) {
+  # TODO: b/399507045 - Cobalt: Fix build errors related to print API's, remove hack
+  if (gtest_enable_absl_printers && is_cobalt &&
```

Comment:
Same here, try to avoid modifying the existing conditional and instead add your own